android/src/main/java/com/facebook/flipper/plugins/databases/DatabasesErrorCodes.java

```diff
@@ -0,0 +1,11 @@
+package com.facebook.flipper.plugins.databases;
+
+public class DatabasesErrorCodes {
```

Comment:
Seems to me like an enum would be a better fit for this.

Reply:
I'm going to keep it this way for now. We can change this in the future.